app/views/documents/fields/_govspeak.html.erb

```diff
@@ -1,3 +1,4 @@
-<%= render "govuk_publishing_components/components/govspeak", {} do %>
-  <%= raw Govspeak::Document.new(@document.contents[schema.id]).to_html %>
-<% end %>
+<%= render "components/markdown_raw", {
+  text: @document.contents[schema.id].to_s,
+  lines: 3
```

Comment:
A small thing, but since we're not rendering govspeak here anymore and instead displaying the raw markdown, should we rename this partial to reflect that?

Reply:
That's a good point - it is just arbitrary text we're passing over. I believe textbox inputs have the same issue when the content is being previewed. @alex-ju do you have an opinion on the name? I was thinking something like 'formatted_text' to go with the use of the <pre> tag.

Reply:
don't have strong opinions on naming. I would be tempted to mimic the label/name of the field, but in our case body is too generic and can refer to many things. `formatted_text ` sounds good.

Reply:
Cool - @emmabeynon that OK with you?

Reply:
Works for me 👍